libs/feature/editor/src/lib/components/record-form/form-field/form-field-online-link-resources/form-field-online-link-resources.component.ts

```diff
@@ -94,7 +114,7 @@ export class FormFieldOnlineLinkResourcesComponent {
             this.valueChange.emit([...this.allResources, newResource])
           }
         },
-        error: (error: Error) => this.handleError(error.message),
+        error: (error: any) => this.handleError(error.error.message),
```

Comment:
This is strange. We should handle an Error here, it seems that we get a nested error in an error instead. It might be worthwhile to figure out where that comes from.

Reply:
It comes from the backend.

So either we change the error object sent by the backend, either we adapt it into the way and form we want it to be on the fly.

Just tell me how you want it to be solved.

Reply:
I would like `handleError` to take in an `Error` object, and the observable to emit an `Error` object in case of exception :)

Reply:
The Error object come from the backend, so how should we proceed ? 

Do we know if the backend always send its errors nested in another Error object ? If that's the case, do we intercept all error object from the backend and reshape them into a proper Error object before treating them ?  :)